src/solver/simulation/common-hydro-levels.cpp

```diff
@@ -33,54 +33,53 @@ void computingHydroLevels(const Data::AreaList& areas,
                           bool remixWasRun,
                           bool computeAnyway)
 {
-    areas.each(
-      [&](const Data::Area& area)
-      {
-          if (!area.hydro.reservoirManagement)
-          {
-              return;
-          }
+    for (const auto& [_, area] : areas)
+    {
+        if (!area->hydro.reservoirManagement)
+        {
+            return;
```

Comment:
Careful, if not in a function you must use keyword `continue` to keep the same behavior.